test/test-start.js

```diff
@@ -0,0 +1,37 @@
+var assert = require("assert"); // assert module is supported from nodejs.
+const execSync = require('child_process').execSync;
+
+describe('Module check', function(){
+	it('Is mongo db installed?', function() {
+		execSync('./third_party/mongodb/bin/mongod -version', function(err, stdout, stderr){

```

Comment:
Is this really sync function? If so, why do we have to pass a callback to receive a result?